velox/vector/ConstantVector.h

```diff
@@ -255,8 +255,12 @@ class ConstantVector final : public SimpleVector<T> {
     return index_;
   }
 
-  void resize(vector_size_t size, bool setNotNull = true) override {
-    BaseVector::length_ = size;
+  void resize(vector_size_t newSize, bool /*setNotNull = true*/) override {
+    BaseVector::length_ = newSize;
+    if constexpr (std::is_same_v<T, StringView>) {
+      SimpleVector<StringView>::resizeIsAsciiIfNotEmpty(
```

Comment:
It is reasonable, but in this context we aren't doing it.
Right now behavior of ConstantVector is the same as of the FlatVector.
We can override all ascii members of SimpleVector in the ConstantVector.

Reply:
Also wonder if it would be better if resize called SimpleVector::resize() , where we have the special case logic for stringview. This way all of this is localized inside SimpleVector.

Reply:
SimpleVector::resize() throws, that's why I didn't touch it.
Probably for purposes that SimpleVector cannot be instantiated by itself.
I'm not against doing it at all - I'm for getting the change in sooner. :)